src/sentry/eventstream/snuba.py

```diff
@@ -217,6 +217,24 @@ def end_delete_tag(self, state):
         state["datetime"] = datetime.now(tz=pytz.utc)
         self._send(state["project_id"], "end_delete_tag", extra_data=(state,), asynchronous=False)
 
+    def tombstone_events(self, project_id, event_ids):
```

Comment:
check if tombstoning before going to the next page woudl invalidate the offset while paginating through group